src/coverlet.core/Coverage.cs

```diff
@@ -300,5 +300,18 @@ private string GetSourceLinkUrl(Dictionary<string, string> sourceLinkDocuments,
             url = sourceLinkDocuments[keyWithBestMatch];
             return url.Replace("*", replacement);
         }
+
+        // This is coming to .NET Standard 2.1 as part of Path
+        private static string GetRelativePath(string relativeTo, string path)
```

Comment:
Move this method into a `PathHelper` class. See diff in this for an example https://github.com/tonerdo/coverlet/compare/remove-netcoreapp-dep?expand=1. That was me taking a stab at fixing this, feel free to ignore the implementation and just follow the structure

Reply:
Add unit tests as well

Reply:
I started to go through unit tests that the corefx team had and adapting them to my code 

https://github.com/dotnet/corefx/blob/a10890f4ffe0fadf090c922578ba0e606ebdd16c/src/System.Runtime.Extensions/tests/System/IO/Path.GetRelativePath.cs

I found that mine didn't handle unix based paths correctly.

The code you had also failed some of those unit tests.

I ended up creating a .netstandard template library which allows the injected template to work in .NET Framework classes.